configuration/restdocs.md

```diff
@@ -7,14 +7,38 @@ title: openHAB REST API
+The REST API provides an interface that can only be accessed programmatically.
+The information available from the API includes Items and their state, installed bindings, sitemap information, and much more. 
+The information exchange takes place through HTTP and the API provides a defined protocoll for information exchange among different computer systems. 
+However, from a technical perspective it is very easy to use. 
+Be aware that the REST API is not automatically installed in openHAB. 
+Access over the Internet to the REST API is possible but this represents a significant security risk, and users are encouraged to [ensure safe and secure connections](http://docs.openhab.org/installation/security.html). 
+
+## Examples for the Use of the REST API
+Some tasks that can be accomplished by using the REST API include:
+
+- Using third party software on cell phones, such as tasker to [open your garage door](https://community.openhab.org/t/triggering-items-using-openhab-2s-rest-api-from-tasker/14027)
+- Interacting with openHAB via other programs; several computer languages offer libraries to easily access REST APIs generally, including openHAB’s REST API
+- Accessing of information from devices that support http communication
+- Updating Item States through external devices that are capable of sending http commands (for example, certain motion detectors or surveillance cameras)
+
+The openHAB REST API frontend is very intuitives and excellently documented and it would vastly exceed this space to present all possible example, nevertheless, here an example to switch an item of the Switch Type (such as a lamp) on and off.
+- Switching ```My_Item``` OFF: 
 
-For further details, please check:
+```java
+curl -X POST --header "Content-Type: text/plain" --header "Accept: application/json" -d "OFF" "http://ip_openHAB_host:8080/rest/items/My_Item"
+``` 
 
-- The interactive REST API documentation available through the openHAB 2 dashboard (Add-on "misc-restdocs", installed by default)
-- [The openHAB 1.x wiki REST API article](https://github.com/openhab/openhab/wiki/REST-API)
+- Switching ```My_Item``` ON: 
 
-{% include contribution-wanted.html %}
+```java
+curl -X POST --header "Content-Type: text/plain" --header "Accept: application/json" -d "ON" "http://ip_openHAB_host:8080/rest/items/My_Item"
+``` 
+
+## Additional Considerations
+One final remark - when openHAB’s REST API is accessed from another domain, the following error may be encountered: [No 'Access-Control-Allow-Origin' header is present on the requested resource](https://community.openhab.org/t/cors-problem-at-rest-api/3712/10). 
+In this case, it may be necessary to add the line org.eclipse.smarthome.cors:enable=true in the file services/runtime.cfg.
+
+The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
```

Comment:
I think it is better to put the server-push capability up at the beginning - please consider this.